build.gradle

```diff
@@ -27,6 +34,9 @@ dependencies {
 
     // to implement producers and consumers
     compileOnly group: 'org.apache.kafka', name: 'connect-api', version: kafkaVersion
+    provided group: 'io.confluent', name: 'kafka-avro-serializer', version: confluentVersion
+
+    provided group: 'org.apache.kafka', name: 'kafka_2.11', version: kafkaVersion
```

Comment:
Replace with compileOnly. Then also the excludejars etc. should not be necessary.